amplipi/streams.py

```diff
@@ -1186,9 +1187,10 @@ def _activate(self, vsrc: int):
       print(f'error starting lms: {exc}')
 
   def _deactivate(self):
-    if self._is_running():
-      self.proc.kill()
-    self.proc = None
+    if self.deallocatable:
```

Comment:
I'm not sure if we want to block this entirely. I'm trying to think through how a soft reset works without this.